contrib/vagrant/provision-full.sh

```diff
@@ -3,8 +3,9 @@ set -euo pipefail
 IFS=$'\n\t'
 USERNAME="${1:-vagrant}"
 
-yum update -y
-yum install -y docker-io git vim golang e2fsprogs tmux httpie ctags hg bind-utils which
+#yum update for fedora provider in vagrant breaks vboxfs 
+#yum update -y
+yum install -y docker-io git vim golang e2fsprogs tmux httpie ctags hg bind-utils which ethtool
```

Comment:
New images have been published for centos/fedora for virtualbox and libvirt.  Please give those a shot.

@rhcarvalho And you are correct that @kargakis's change should now be in there.